src/static/current-state/components/QueueItem.tsx

```diff
@@ -320,6 +322,8 @@ export class QueueItem extends React.Component<QueueItemProps, QueueItemState> {
       request: { dependsOn, pullRequestId, pullRequest },
     } = status;
 
+    const buildId = status.request.buildId;
```

Comment:
Huh... I could have sworn the reason we didn't have the build url in the title to begin with was that we didn't have the build status's info for every item in the queue, we fetched it one at a time as you clicked "show more".

Is my memory of this just wrong or did this change at some point? Do you recall @itsjackgardner ?

Reply:
We seem to, as far as I've observed 😅